python/pyspark/ml/param/__init__.py

```diff
@@ -134,6 +134,16 @@ def toListFloat(value):
                 return [float(v) for v in value]
         raise TypeError("Could not convert %s to list of floats" % value)
 
+    @staticmethod
+    def toListListFloat(value):
```

Comment:
We need a test case in `ParamTypeConversionTests` for this new method; see `test_list_float` for reference.